thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/template.hbs

```diff
@@ -146,81 +145,50 @@
     </div>
 
     {{!-- Field: Sensitivity with mode option --}}
-    {{#if isNewUx}}
-      <div class="te-form__vertical-block col-xs-12">
-        <label for="select-sensitivity" class="control-label te-label te-label--taller required">Sensitivity (optional)</label>
-        <div class="te-form__indent-block--custom">
-          <span class="te-form__list-label">When</span>
-          <div class="te-form__tune-field">
-            {{#power-select
-              triggerId="alert-tune-severity"
-              triggerClass="te-form__tune-field"
-              verticalPosition="below"
-              placeholder="Select a severity"
-              options=tuneSeverityOptions
-              searchEnabled=false
-              selected=selectedSeverityOption
-              onchange=(action (mut selectedSeverityOption))
-              disabled=(not isMetricSelected)
-              as |severity|
-            }}
-              {{severity}}
-            {{/power-select}}
-          </div>
-          <span class="te-form__list-label">is >=</span>
-          <div class="te-form__tune-field te-form__tune-field--amount">
-            {{input
-              type="text"
-              id="custom-tune-percent"
-              class="form-control te-input te-input"
-              value=customPercentChange
-              disabled=(not isMetricSelected)
-            }}
-          </div>
-          <span class="te-form__list-label--suffix">{{sensitivityUnits}}, </span>
-          <span class="te-form__list-label">MTTD should be no more than</span>
-          <div class="te-form__tune-field te-form__tune-field--amount">
-            {{input
-              type="text"
-              id="custom-tune-mttd"
-              class="form-control te-input"
-              value=customMttdChange
-              disabled=(not isMetricSelected)
-            }}
-          </div>
-          <span class="te-form__list-label">hours</span>
+    <div class="te-form__vertical-block col-xs-12">
+      <label for="select-sensitivity" class="control-label te-label te-label--taller required">Sensitivity (optional)</label>
+      <div class="te-form__indent-block--custom">
+        <span class="te-form__list-label">When</span>
+        <div class="te-form__tune-field">
+          {{#power-select
+            triggerId="alert-tune-severity"
+            triggerClass="te-form__tune-field"
+            verticalPosition="below"
+            placeholder="Select a severity"
+            options=tuneSeverityOptions
+            searchEnabled=false
+            selected=selectedSeverityOption
+            onchange=(action (mut selectedSeverityOption))
+            disabled=(not isMetricSelected)
+            as |severity|
+          }}
+            {{severity}}
+          {{/power-select}}
         </div>
+        <span class="te-form__list-label">is >=</span>
```

Comment:
Let's use `<label for="...">` here to be more semantically correct (since we have an `input`)
Applies to all label + input

Reply:
done.